test/PaymasterCommitment.test.ts

```diff
@@ -207,16 +280,19 @@ contract('Paymaster Commitment', function ([_, relayOwner, relayManager, relayWo
 
       }, sharedRelayRequestData, chainId, forwarderInstance)
 
-      const gasLimits = await paymasterContract.getGasLimits()
+      const gasLimits = await paymasterContract.getGasAndDataLimits()
+      const relayCall = toBuffer(relayHubInstance.contract.methods.relayCall(1, r.req, r.sig, '0x', externalGasLimit).encodeABI())
+      const dataGasCost = await relayHubInstance.calldataGasCost(relayCall.length)
+      const maxRelayExposure = parseInt(gasLimits.acceptanceBudget) + dataGasCost.toNumber()
       // fail if a bit lower
-      expectRevert(relayHubInstance.relayCall(parseInt(gasLimits.acceptanceBudget) - 1, r.req, r.sig, '0x', externalGasLimit, {
+      expectRevert(relayHubInstance.relayCall(maxRelayExposure - 1, r.req, r.sig, '0x', externalGasLimit, {
         from: relayWorker,
         gas: externalGasLimit,
         gasPrice
-      }), 'unexpected high acceptanceBudget')
+      }), 'unexpected high maxRelayExposure')
```

Comment:
as mentioned earlier, this is not exposure that is high, that is call data size/ low Paymaster acceptanceBudget. Now this is an interesting test to add - raise this exception on call data size.

Reply:
fixed